pages/advanced/api-service-architecture/safe-transaction-service/interacting-with-api.md

```diff
@@ -0,0 +1,576 @@
+    -H 'accept: application/json' \
+    -H 'Content-Type: application/json' \
+    -d '{
+        "safe": "0xc62C5cbB964459F3C984682f78A4d3ffffffffff",
+        "to": "0x795D6C88B4Ea3CCffffffffffCa8a11Bc0496228",
+        "value": 2000000000000000,
+        "data": null,
+        "operation": 0,
+        "gasToken": "0x0000000000000000000000000000000000000000",
+        "safeTxGas": 0,
+        "baseGas": 0,
+        "gasPrice": 0,
+        "refundReceiver": "0x0000000000000000000000000000000000000000",
+        "nonce": 15,
+        "contractTransactionHash": "0x56b2931d1053b6afffffffffff3ba29b5c2baafdf1a588850da72a62674941b6",
+        "sender": "0xAA86E576c084aCFa56fc4D0E17967ffffffffff8",
+        "signature": "0x6a2b57023af16241511619ea95f7cd03d00aa6b79d1ca80e21a0b89cd2c38ffffffffff9b738ffbd680c4d717b9b0c9eae568f3edebc40a0c004700bffffffffff"
+    }'
+```
+
+- Add signatures to transaction
+
+```bash
+    curl -X 'POST' \
+    'https://safe-transaction-sepolia.safe.global/api/v1/multisig-transactions/0x56b2931ffffffffff303bbaadf3ba29b5c2baafdf1a5ffffffffff62674941b6/confirmations/' \
+    -H 'accept: application/json' \
+    -H 'Content-Type: application/json' \
+    -d '{
+        "signature": "0x8c41aaa029db2942a1574c3ec9442b80764e953f21f994edb641d33ffffffffffa55ffffffffffefcaea5b3360bd4c789803fe289b2ad467fc9b3bedb479dad20"
+    }'
+```
+
+#### Using safe-core-sdk
+
+```js
+    async function createTransactionAndSignByOwners() {    
+        // Instantiate an EthAdapter for Owner A
+        const ethProvider = new ethers.JsonRpcProvider(config.RPC_URL);
+
+        const ownerA = new ethers.Wallet(config.OWNER_A_PRIVATE_KEY, ethProvider);
+        const ethAdapterOwnerA = new EthersAdapter({
+            ethers,
+            signerOrProvider: ownerA
+        });
+
+        // Initialize the Protocol Kit for Owner A
+        const protocolKitOwnerA = await Safe.create({
+            ethAdapter: ethAdapterOwnerA,
+            safeAddress: config.SAFE_ADDRESS
+        });
+
+        // Create Tx
+        const safeTransactionData: MetaTransactionData = {
+            to: config.TO,
+            value: config.VALUE,
+            data: '0x',
+            operation: OperationType.Call
+        };
+
+        const safeTransaction = await protocolKitOwnerA.createTransaction({
+            transactions: [safeTransactionData]
+        });
+
+        console.log("Tx created: %O", safeTransaction)
+
+        // Sign Tx by Owner A
+        const safeTxHash = await protocolKitOwnerA.getTransactionHash(safeTransaction);
+        const signatureOwnerA = await protocolKitOwnerA.signHash(safeTxHash);
+
+        console.log("Tx signed by Owner A: %s", signatureOwnerA.data)
+        
+        // Initialize the Api Kit
+        const apiKit = new SafeApiKit({
+            chainId: 11155111n,
+            txServiceUrl: 'https://safe-transaction-sepolia.safe.global/api'
```

Comment:
`txServiceUrl` is an optional parameter for chains that are supported by the production transaction service.

Reply:
Yes, I know, I just thought it was interesting to put it explicitly in the example in case you want to use chains that are not in production. What do you think?